helm-charts/basehub/Chart.yaml

```
@@ -25,7 +25,7 @@ dependencies:
     version: "2024.1.0"
     repository: "https://helm.dask.org/"
     condition: dask-gateway.enabled
-  - name: jupyter-home-nfs
+  - name: jupyterhub-home-nfs
     version: 0.0.5
```

The `0.0.5` version isn't ported over to the new package under 2i2c Github org. I just pushed `0.0.6` tag to get a new version to use. Let's use that here.
```suggestion
    version: 0.0.6
```